attributes/default.rb

```diff
@@ -130,8 +130,30 @@
   default['kibana']['kibana_service'] = "http://#{node['kibana']['interface']}:#{node['kibana']['port']}"
 end
 
-#<> The kibana service configuration source
-default['kibana']['service']['source'] = 'upstart.conf.erb'
+# kibana service configurations - defaults to settings for Ubuntu 14.04
+case node['platform']
+when 'centos'
+  if node['platform_version'] < '6.9'
+    default['kibana']['service']['provider'] = Chef::Provider::Service::Init::Redhat
```

Comment:
You are correct - I just pushed a commit which removes this to clean up and reduce code.  Thank you!